test/integration/Elsa.Core.IntegrationTests/Persistence/EntityFramework/EntityFrameworkIntegrationTests.cs

```diff
@@ -12,17 +12,17 @@
 using Elsa.Persistence.EntityFramework.Core;
 using Microsoft.EntityFrameworkCore.Internal;
 using Moq;
+using Elsa.Testing.Shared.Helpers;
 
 // ReSharper disable EF1001
 namespace Elsa.Core.IntegrationTests.Persistence.EntityFramework
 {
     public class EntityFrameworkIntegrationTests
     {
         [Theory(DisplayName = "A persistable workflow instance with default persistence behaviour should be persisted-to and readable-from an Entity Framework store after being run"), AutoMoqData]
-        public async Task APersistableWorkflowInstanceWithDefaultPersistenceBehaviourShouldBeRoundTrippable(
-            [HostBuilderWithPersistableWorkflowAndEfSqlite]
-            IHostBuilder hostBuilder)
+        public async Task APersistableWorkflowInstanceWithDefaultPersistanceBehaviourShouldBeRoundTrippable([WithPersistableWorkflow,WithEntityFramework] ElsaHostBuilderBuilder hostBuilderBuilder)
```

Comment:
Should it be "Persistence" instead of "Persistance"?

Reply:
It should!  And now it is!